spec/features/editing_images/edit_image_crop_publishing_api_down_spec.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+RSpec.feature "Edit image crop when Publishing API is down", js: true do
+  scenario do
+    given_there_is_a_document_with_images
+    when_i_visit_the_lead_images_page
+    and_the_publishing_api_is_down
+    and_i_crop_the_image
+    then_i_see_the_image_is_updated
+    and_the_preview_creation_failed
+  end
+
+  def given_there_is_a_document_with_images
+    document_type_schema = build(:document_type_schema, lead_image: true)
+    document = create(:document, document_type: document_type_schema.id)
+    create(:image, :in_asset_manager, document: document)
```

Comment:
Do you think this would be clearer if you explicitly specified the file used? I was wondering if the drag actually tested anything and so had to dig into the factories to work out we were defaulting to a 1000x1000 image.

Reply:
Sure - I've enhanced it a bit more to make it clearer what the crop was and how it's changed.